packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -35,36 +35,22 @@ import 'multidex.dart';
 /// The directory where the APK artifact is generated.
 Directory getApkDirectory(FlutterProject project) {
   return project.isModule
-    ? project.android.buildDirectory
```

Comment:
Please revert all formatting changes here, per https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#formatting. It is difficult to review the diff when formatting changes are included, and we would not be merge the style changes anyway, as they violate our style guide. Thanks.